ketcher-autotests/tests/Settings/General/default-settings-verification.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { test } from '@playwright/test';
+import {
+  selectTopPanelButton,
+  TopPanelButton,
+  takeEditorScreenshot,
+} from '@utils';
+
+test.describe('Open Ketcher', () => {
```

Comment:
please use in describe more meaningful description regarding this particular test file) for example, "Ketcher settings panel" etc